iaso/api/pages.py

```diff
@@ -65,5 +64,10 @@ def get_object(self):
     def get_queryset(self):
         user = self.request.user
         order = self.request.query_params.get("order", "created_at").split(",")
-        users = User.objects.filter(iaso_profile__account=user.iaso_profile.account)
-        return Page.objects.filter(users__in=users).order_by(*order).distinct()
+        
+        queryset = Page.objects.filter(account__users=user.iaso_profile.account.users)
```

Comment:
Why fetch all users from the account instead of directly checking on the account?

Reply:
Because each embedded link must be linked to one or multiple users in the account. So when we filter only on account it will get also embedded links which are not linked to any user. So that why I filtered on users in the account.